docs/thirdparty-openapi3-snippets.yaml

```diff
@@ -38,8 +52,19 @@ components:
         The auth channel being used for the consentRequest.
         - "WEB" - The Web auth channel
         - "OTP" - The OTP auth channel
-    AuthScopesType:
-      title: AuthScopesType
+    AuthorizationResponseType:
+      title: AuthorizationResponseType
+      description: |
+        Enum containing response information; if the customer entered the
+        authentication value, rejected the transaction, or requested a
+        resend of the authentication value.
+      type: string
+      enum:
+        - ENTERED
+        - REJECTED
+        - RESEND
+    AuthorizationScopeType:
```

Comment:
Maybe to avoid confusion we could call this `ConsentScopeType`

Reply:
sounds good done.